test/ui-testing/publishButton.test.js

```diff
@@ -1,14 +1,24 @@
 const timeout = process.env.SLOWMO ? 60000 : 10000;
 const fs = require('fs');
-beforeAll(async () => {
-  path = fs.realpathSync('file://../examples/index.html');
-  await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
-});
 
 describe('Publish button', () => {
-  
-  test('something we are testing described here', () => {
+  var $;
+  beforeAll(async () => {
+    path = fs.realpathSync('file://../examples/index.html');
```

Comment:
I'm not 100% sure here, but working from https://stackoverflow.com/a/46987959/1116657 I'm thinking we should try something like this:

```suggestion
import { launch } from 'puppeteer'
(async () => {
    const browser = await launch({headless: false});
    const page = await browser.newPage();
    await page.goto('file://../examples/index.html', {waitUntil: 'networkidle'});
    //path = fs.realpathSync('file://../examples/index.html');
    await page.addScriptTag({url: 'https://code.jquery.com/jquery-3.2.1.min.js'});
    await page.waitForNavigation()
```

My main uncertainty is how to load the local file rather than a remote URL. Not sure what I wrote here will work.

Reply:
Yea exactly, I bumped into this as well since I decided I could no longer make use of jsdom. I tried this but it didn't work for me.
I think we might be able to load the local version of jquery using:
```
await page.addScriptTag({path: require.resolve('jquery')})
```

but I wasn't sure what that path (path to jquery) is for our project. So I did 
```
const jqueryPath = require('jquery')
```
then I hovered on 'jquery' and saw the path, I copied it and used it directly. Here's the new error I got:

![Screenshot 2021-12-19 at 19 17 08](https://user-images.githubusercontent.com/49737323/146687480-b9302384-30fd-4456-b37c-402a35beabec.png)

I'm not so sure what the '_productName' is but I'm still looking. Do you have any idea what it is?
@jywarren